### ISSUE TITLE
Text file import/export representation

### ISSUE THREAD

**Issue by bslund** [hyperborea/heimdall#5]
User story:
As a Heimdall user I want to manage my sources, jobs, visualisations and dashboards through a concise text file format so that I can better manage my dashboards.

**Comment by hyperborea** [comment]
Do I understand correctly that this is about the configuration (e.g. the job form including the query)  and not the results (i.e. the actual data retrieved or visualization generated)? What's the use case?

**Comment by bslund** [comment]
Exactly, the use case is to manage a large number of jobs in an efficient manner.
